product-mini/platforms/zephyr/simple-http/wasm-apps/http_get.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <errno.h>
+#include <string.h>
+
+#ifdef __wasi__
+#include <wasi_socket_ext.h>
+#endif
+
+/* HTTP server to connect to */
+#define HTTP_HOST "192.0.2.10"
+/* Port to connect to, as string */
+#define HTTP_PORT "8000"
+/* HTTP path to request */
+#define HTTP_PATH "/"
+
+#define SSTRLEN(s) (sizeof(s) - 1)
+// #define CHECK(r) { if (r == -1) { printf("Error %d: " #r "\n", errno);
+// exit(1); } }
+
+#define REQUEST "GET " HTTP_PATH " HTTP/1.0\r\nHost: " HTTP_HOST "\r\n\r\n"
+
+static char response[1024];
+
+int
+main(int argc, char **argv)
+{
+    int st, sock;
+    struct sockaddr_in addr;
+    int rc = 0;
+
+    printf("[wasm-mod] Preparing HTTP GET request for http://" HTTP_HOST
+           ":" HTTP_PORT HTTP_PATH "\n");
+
+    memset(&addr, 0, sizeof(addr));
+    addr.sin_family = AF_INET;
+    addr.sin_port = htons(8000);
+    addr.sin_addr.s_addr =
+        htonl(3221225994); // hard coded IP address for 192.0.2.10
```

Comment:
I propose to use the hex notation here instead: `htonl(0xC000020A)`. It works the same but is easier to read and to modify

Reply:
done